project/docker-compose.yml

```
@@ -0,0 +1,88 @@
+        required: true
+    networks:
+      - default
+    restart: unless-stopped
+
+  postgres:
+    image: docker.io/postgres:17-alpine
+    networks:
+      - default
+    restart: unless-stopped
+    healthcheck:
+      test: [ "CMD", "pg_isready", "--dbname=app" ]
```

YAMLlint (1.38.0)

[error] too many spaces inside brackets

(brackets)